x-pack/plugin/esql/qa/testFixtures/src/main/resources/spatial.csv-spec

```diff
@@ -1065,8 +1065,8 @@ FROM airports_web
 | STATS centroid=ST_CENTROID_AGG(location), count=COUNT()
 ;
 
-centroid:cartesian_point                      | count:long
-POINT(-266681.67563861894 3053301.5120195406) | 849
+centroid:cartesian_point                     | count:long
+POINT(-266681.66530554957 3053301.506061676) | 849
```

Comment:
@craigtaverner somehow switching to a copy of the `AggregateFunction`/`SpatialAggregateFunction` classes leads to different rounding here.

1. Maybe we should find a way to round `POINT`s in csv tests, as this may not be the last time we need to update all of the `st_centroid_agg` tests because of rounding errors.
2. Most rounding errors are more than 2 decimal points after the 0, but in some cases, like here, already the 2nd decimal after the 0 is affected. Is that expected?

Reply:
I found out the rounding errors were due to accidentally disabling the `SpatialDocValuesExtraction` optimization rule, because it was matching on `ql`'s `SpatialAggregateFunction` instead of the esql clone of this class.

@craigtaverner , I think no tests failed due to this - I think we need to add a test to the `LocalPhysicalPlanOptimizerTests`, or is there some test already that _should_ have caught this?

Reply:
False alert, this is tested nicely in the (non-local) `PhysicalPlanOptimizerTests`.